content/en/docs/running.md

```diff
@@ -1,156 +1,155 @@
 ---
-title: Running Falco
-weight: 2
+title: Running 
+description: Operating and Managing Falco
+weight: 4
 ---
 
-Falco is meant to be run as a service. But for experimentation and designing/testing rulesets, you will likely want to run it manually from the command-line.
 
-## Running Falco as a service
+## Run Falco as a service
 
-Once you've [installed](../installation) Falco as a package, you can start the service:
+If you installed Falco by using [the deb or the rpm](../installation) package, you can start the service:
 
 ```bash
 service falco start
 ```
 
-The default configuration logs events to syslog.
+You can also view the Falco logs using `journalctl`.
 
-## Reloading configuration
-
-As of Falco >= 0.13.0, on SIGHUP Falco will fully restart its main loop, closing the device for the kernel module and re-reading all config, etc. from scratch. This can be useful if you want to change the set of rules files, config, etc. on the fly without having to restart Falco.
-
-## Running Falco in a container
-
-The current version of Falco is available as the `falcosecurity/falco:{{< latest >}}` container. Here's an example command to run the container locally on Linux:
-
-```bash
-docker run \
-  --interactive \
-  --privileged \
-  --tty \
-  --name falco \
-  --volume /var/run/docker.sock:/host/var/run/docker.sock \
-  --volume /dev:/host/dev \
-  --volume /proc:/host/proc:ro \
-  --volume /boot:/host/boot:ro \
-  --volume /lib/modules:/host/lib/modules:ro \
-  --volume /usr:/host/usr:ro \
-  falcosecurity/falco:{{< latest >}}
-```
-
-By default, starting the container will attempt to load and/or build the Falco kernel module. If you already know that the kernel module is loaded and want to skip this step, you can set the environment variable `SYSDIG_SKIP_LOAD` to `1`:
-
-```bash
-docker run ... -e SYSDIG_SKIP_LOAD=1 ... falcosecurity/falco:{{< latest >}}
-```
-
-## Running Falco in a Kind cluster
-
-The easiest way to run Falco on a [Kind](https://github.com/kubernetes-sigs/kind) cluster is as follows:
-
-1. Create a configuration file. For example: `kind-config.yaml`
-
-2. Add the following to the file:
-```yaml
-kind: Cluster
-apiVersion: kind.x-k8s.io/v1alpha4
-nodes:
-- role: control-plane
-  extraMounts:
-  - hostPath: /dev
-    containerPath: /dev
-```
-
-3. Create the cluster by specifying the configuration file:
+```bash 
+journalctl -fu falco
 ```
-kind create cluster --config=./kind-config.yaml
-```
-
-4. [Install](../installation) Falco in your Kubernetes cluster with kind.
 
-## Running Falco manually
+## Run Falco manually
 
-If you'd like to run Falco by hand, here's the full usage description for falco:
+If you'd like to run Falco by hand, here's the full usage description for Falco:
```

Comment:
I generally do not suggest adding things like this to the documentation. This means every time we change something in the code we have to remember to change it here. Also this isn't relevant to all versions of Falco.

I would rather see something like:

The Falco help menu that is specific to your version of Falco can be found by

```
falco --help
```